.github/workflows/pr.yaml

```diff
@@ -30,21 +32,18 @@ jobs:
     strategy:
       matrix:
         linter-version: [KnownGoodVersion, Latest]
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Locate trunk
```

Comment:
This is necessary (or something similar) to make sure trunk is installed on the CI machine

Reply:
See below, I changed it to use the trunk launcher via npm